server/services/componentService.test.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '../data/model/component'
+import ComponentClient, { AvailableComponent } from '../data/componentClient'
+import ComponentService from './componentService'
+
+describe('Component Service', () => {
```

Comment:
(Unrelated to code change - just added to increase code coverage and make the pipeline happy)